docs/tools/firmware.sh

```diff
@@ -124,8 +124,8 @@ create_rpm () {
 	then
 		nvramfile=$(ioreg -l | grep RequestedFiles | cut -d "/" -f 5 | rev | cut -c 4- | rev)
 		txcapblob=$(ioreg -l | grep RequestedFiles | cut -d "/" -f 3 | cut -d "\"" -f 1)
-		cp ${verbose} /usr/share/firmware/wifi/C-4364__s-B2/${nvramfile} "$HOME/rpmbuild/BUILD/brcmfmac4364b2-pcie.txt"
-		cp ${verbose} /usr/share/firmware/wifi/C-4364__s-B2/${txcapblob} "$HOME/rpmbuild/BUILD/brcmfmac4364b2-pcie.txcap_blob"
+		tar --append ${verbose} -f $HOME/rpmbuild/SOURCES/firmware.tar /usr/share/firmware/wifi/C-4364__s-B2/${nvramfile}
```

Comment:
Directly appending to tar wont rename the file. You'll have to copy.

Reply:
I fixed it